src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.vb

```diff
@@ -0,0 +1,459 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DynamicInvocationExpression_Basic()
```

Comment:
>DynamicInvocationExpression_Basic [](start = 19, length = 33)

Please add test for indexing of off an object and invoking a delegate with object as an argument (I think you have similar scenario for C#).

Reply:
This comment was marked as resolved, but I do not see a test that covers "invoking a delegate with object as an argument". Sorry if the comment was confusing, it was talking about two different scenarios. 

---
In reply to: [137896866](https://github.com/dotnet/roslyn/pull/21711#discussion_r137896866) [](ancestors = 137896866)

Reply:
@AlekseyTs I added [DynamicInvocationExpression_InvokeDelegateWithArgument](https://github.com/dotnet/roslyn/pull/21711/files#diff-23d389dbc449a97330407195ee807219R435). If this is not the scenario you were referring to, can you please provide a small code sample for your requested test?

Reply:
> I added DynamicInvocationExpression_InvokeDelegateWithArgument. If this is not the scenario you were referring to, can you please provide a small code sample for your requested test?

The test always indexes of off an Object, never of off a delegate type. None of the IOperation nodes are typed as a delegate type. I am thinking about simple scenario ```d(o)```, where ```d``` is statically typed as delegate and ```o``` is typed as object.